GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2454,7 +2454,7 @@ private void RevisionInfo_CommandClicked(object sender, ResourceManager.CommandE
 
                     if (found)
                     {
-                        found = RevisionGrid.SetSelectedRevision(revision);
+                        found = RevisionGrid.SetSelectedRevision(revision, showNoRevisionMsg: !AppSettings.ShowFirstParent);
```

Comment:
I think there may be other reasons why a given revision isn't visible, i.e. other filters are applied.

Reply:
The intention is that the message should be interpreted for any filter not just "advanced filter", branchfilters etc but also show first parents etc.
Clarification?

Reply:
Sorry, the meaning of your message is still escaping me... 

We should avoid using `AppSettings.ShowFirstParent` and other filtering options on their own. All filtering decisions or checks should be done only via `FilterInfo` object (which is essentially an abstraction layer).